contents/staticAnalysis/ESLint.md

```diff
@@ -0,0 +1,130 @@
+# ESLint
+
+Author: [Nicholas Chua](https://github.com/nus-cs3281/2018/blob/master/students/nicholasChua/nicholasChua-Resume.md)
+
+## Overview
+
+[ESLint](https://eslint.org/docs/about/) is an open source JavaScript linter.
+
+Code linting is a type of static analysis that helps find problematic patterns or code that does not follow a certain style guideline.
+
+JavaScript, being a loosely-typed, interpreted language, is prone to developer errors and therefore linting is especially important when working with JavaScript.
+
+## Features

```

Comment:
This section seems a bit redundant as the same things are mentioned again later. May be add a paragraph to the `Overview` section that mentions there are built-in rules and can be customized further via custom rules.